src/Chapter3.hs

```diff
@@ -344,6 +344,17 @@ of a book, but you are not limited only by the book properties we described.
 Create your own book type of your dreams!
 -}
 
+-- In descriptions you provide MkKnight constructor, but in solutions you have
+-- just Book constructor. Why??
```

Comment:
Constructors can have the same name as the data type. We add the `Mk` prefix to help people understand the difference between types and constructors. Otherwise, it may be very confusing. However, in practice, when using Haskell in real-world, most people don't bother with adding `Mk` (extra two letters to write!) and always name constructors using the same name as the type itself.